index.js

```diff
@@ -130,24 +130,8 @@ var bundleWithBrowserify = function(fn) {
     entryCache
   ];
 
-  var entrySources = {};
-
-  var resolveSources = function resolveSources(key) {
-    entrySources[key] = true;
-
-    for (var depPath in sources[key][1]) {
-      var depKey = sources[key][1][depPath];
-
-      if (!entrySources[depKey]) {
-        resolveSources(depKey);
-      }
-    }
-  };
-
-  resolveSources(entryKey);
-
   return '(' + bundleFn + ')({'
-        + Object.keys(entrySources).map(function(key) {
+        + Object.keys(sources).map(function(key) {
```

Comment:
doesn't this then bundle all the sources rather than the sources based on the entry and all its deps?

Reply:
That is correct. That is actually the current behavior of webworkify 1.0.2 (which contrib-hls was locked at before webwackify) as well as the webpack half of webwackify. I based the browsierify half of webwackify on webworkify 1.5.0 without realizing contrib-hls was locked at 1.0.2. As far as I'm aware, there doesn't seem to be much of a downside to including all the sources in the worker blob, even if it doesn't need them

Reply:
OK, sounds good.